app/views/videos/show.html.haml

```diff
@@ -6,8 +6,39 @@
       .video_info.col-sm-3
         %header
           %h3= @video.title
-          %span Rating: 4.5/5.0
+          %span Rating: #{@video.average_rating}/5.0
         %p= @video.description
         .actions
           %a.btn.btn-primary(href="") Watch Now
-          %a.btn.btn-default(href="") + My Queue
\ No newline at end of file
+          %a.btn.btn-default(href="") + My Queue
+%section.reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_for [@video, Review.new] do |f|
+        %fieldset
+          .form-group
+            %label Rate this video
+            .row
+              .col-sm-3
+                = f.select :rating, options_for_select([5,4,3,2,1].map {|rating| [rating.to_s + " Star".pluralize(rating), rating]},["5 Stars", 5])

```

Comment:
You can simplify the array produced by `.map` as `pluralize` is able to accept 2 arguments.

```ruby
[pluralize(rating, "Star"), rating]
```